openwisp_notifications/tests/test_notifications.py

```diff
@@ -60,7 +60,6 @@ def setUp(self):
         self.notification_options = dict(
             sender=self.admin,
             description='Test Notification',
-            level='info',
```

Comment:
I went ahead and fixed the reason for failing test. This would allow us to close this PR quickly. 

In previous commits, we changed the logic of the `notify` handler to give preference to the `level` kwarg passed to the notify signal. 

@Dhanus3133 we need to add a test case in test_models.py which verifies that the preference of level is given to the `level` kwargs over the `level` defined in notification type.

Reply:
I believe the file is `test_notifications.py`. Updated with the changes there.